test-requirements.txt

```diff
@@ -10,3 +10,5 @@ fpdf2==2.8.2
 papermill==2.6.0
 pyfakefs==5.8.0
 torch==2.4.1
+boto3
+moto
```

Comment:
please set specific versions here, to avoid breakages when those libraries get new releases

Reply:
Thanks, done